console/frontend/src/main/frontend/src/app/views/webservices/webservices.component.html

```diff
@@ -21,7 +21,7 @@ <h4>Available REST Services</h4>
                 <tr *ngFor="let service of services; index as index">
                   <td>{{ index }}</td>
                   <td>{{ service.name }}</td>
-                  <td>{{ service.method }}</td>
+                  <td>{{ service.methods }}</td>
```

Comment:
A small improvement could be to add a space after the comma.

That can be achieved like this:

```suggestion
                  <td>{{ service.methods.join(', ') }}</td>
```